libogc/console_internal.h

```diff
@@ -15,11 +28,12 @@ typedef struct _console_data_s {
 	unsigned char *font;
 	int con_xres,con_yres,con_stride;
 	int target_x,target_y, tgt_stride;
-	int cursor_row,cursor_col;
-	int saved_row,saved_col;
+	int cursorX,cursorY;
+	int prevCursorX,prevCursorY;
 	int con_rows, con_cols;
-
-	unsigned int foreground,background;
+	int tabSize;
+	unsigned int fg,bg;
```

Comment:
i saw its like that in libcitru as wel, but kinda sad that everything is named nicely except foreground & background. they just get fg/bg